examples/multiple_windows/lib/app/main_window.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: implementation_imports
+
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/_window.dart';
+
+import 'window_content.dart';
+
+import 'regular_window_content.dart';
+import 'window_settings_dialog.dart';
+import 'models.dart';
+import 'regular_window_edit_dialog.dart';
+
+class MainWindow extends StatelessWidget {
+  MainWindow({super.key, required BaseWindowController mainController}) {
+    windowManager.add(
+      KeyedWindow(
+        isMainWindow: true,
+        key: UniqueKey(),
+        controller: mainController,
+      ),
+    );
+  }
+
+  final WindowManager windowManager = WindowManager();
+  final WindowSettings settings = WindowSettings();
+
+  @override
+  Widget build(BuildContext context) {
+    return WindowManagerAccessor(
+      windowManager: windowManager,
+      child: WindowSettingsAccessor(
+        windowSettings: settings,
+        child: ViewAnchor(
+          view: ListenableBuilder(
+            listenable: windowManager,
+            builder: (BuildContext context, Widget? _) {
+              final List<Widget> childViews = <Widget>[];
+              for (final KeyedWindow window in windowManager.windows) {
+                if (window.parent == null && !window.isMainWindow) {
```

Comment:
Could you add a comment explaining which types of windows this `if` is trying to target?